tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "compilerOptions": {
-    "allowJs": true,
+    "declaration": true,
     "allowSyntheticDefaultImports": true,
-    "checkJs": false,
```

Comment:
`declaration` doesn't work with `checkJs`/`allowJs`. It does work in the new version of TS, however I ran into issues when trying to upgrade TS, and it looks like these compiler options are not needed anyway. To verify that, I compared the list of generated `dist` files before/after this change (`ls src/**/dist/**/*.js`), and the results were the same.